### ISSUE TITLE
Does not work with androidx.appcompat:appcompat:1.2.0

### ISSUE THREAD

**Issue by alexeyvasilyev** [franmontiel/LocaleChanger#19]
Tested on Android 11.
Related issue https://github.com/zeugma-solutions/locale-helper-android/issues/15 and https://github.com/akexorcist/Localization/issues/64

**Comment by AhmedDonkl** [comment]
@alexeyvasilyev Do you find another way to solve this?

**Comment by alexeyvasilyev** [comment]
Switched to locale-helper-android latest beta. It works at least partitially.

**Comment by g-mac** [comment]
I am also having this issue.. can we expect a fix for this? Or do I also have look into switching libraries? Thanks =)

**Comment by franmontiel** [comment]
I've just published v1.1 fixing the problem. 
It is required that you **remove** the following piece of code from your Activities:
```java
@Override
protected void attachBaseContext(Context newBase) {
    newBase = LocaleChanger.configureBaseContext(newBase);
    super.attachBaseContext(newBase);
}
```
And **replace it** by this piece of code:
```java
private LocaleChangerAppCompatDelegate localeChangerAppCompatDelegate;

@NonNull
@Override
public AppCompatDelegate getDelegate() {
    if (localeChangerAppCompatDelegate == null) {
            localeChangerAppCompatDelegate = new LocaleChangerAppCompatDelegate(super.getDelegate());
    }

    return localeChangerAppCompatDelegate;
}
```

**Comment by emretekin** [comment]
> I've just published v1.1 fixing the problem.
> It is required that you **remove** the following piece of code from your Activities:
> 
> ```java
> @Override
> protected void attachBaseContext(Context newBase) {
>     newBase = LocaleChanger.configureBaseContext(newBase);
>     super.attachBaseContext(newBase);
> }
> ```
> 
> And **replace it** by this piece of code:
> 
> ```java
> private LocaleChangerAppCompatDelegate localeChangerAppCompatDelegate;
> 
> @NonNull
> @Override
> public AppCompatDelegate getDelegate() {
>     if (localeChangerAppCompatDelegate == null) {
>             localeChangerAppCompatDelegate = new LocaleChangerAppCompatDelegate(super.getDelegate());
>     }
> 
>     return localeChangerAppCompatDelegate;
> }
> ```

This is works for me! 
Thanks

**Comment by g-mac** [comment]
Worked for me!!! Thank you so much!
